docs/production-setup.md

```diff
@@ -88,6 +88,12 @@ fvm flutter build ipa --flavor Bayern --release --dart-define=environment=produc
 
 
 ### Backend and Administration
+
+#### Important: 
```

Comment:
please mind this @ztefanie , @seluianova, @steffenkleinle if you add any values to the backend `config.yml`

Reply:
I don't have an access to the link below, can you grant it to me?

Reply:
ok, that is ugyl :cry: 

I think this hint should be in the config file in a comment, because then, when changing the file you get the reminder.

I created a ticket to solve this issue: https://github.com/digitalfabrik/entitlementcard/issues/1627

Reply:
oh sure i moved the comment to the `config.yml`